packages/mobile/src/identity/verification.ts

```diff
@@ -324,9 +322,9 @@ export function* doVerificationFlow(withoutRevealing: boolean = false) {
             phoneHashDetails,
             attestations
           )
-          ValoraAnalytics.track(VerificationEvents.verification_reveal_all_attestations_complete)
         }
       }
+      ValoraAnalytics.track(VerificationEvents.verification_reveal_all_attestations_complete)
```

Comment:
I would put inside `if (!withoutRevealing)` block, cause otherwise we would fire `verification_reveal_all_attestations_complete` even when there was no `verification_reveal_all_attestations_start` before.

Reply:
oh good call 👍